test/LPStaking.test.js

```diff
@@ -22,23 +22,31 @@ contract("LPStaking", accounts => {
         token = await Token.deployed();
         lp_token = await LPToken.new();
 
-        factory = await LPFactory.deployed();
+        factory = await StakingFactory.deployed();
     })
 
     it("check token", async () => {
         let balance = await token.balanceOf(owner);
 
         expect(balance.toString()).to.equal(
-            web3.utils.toWei('20000000'));
+            web3.utils.toWei('1000000000'));
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/tranche-jibrel/new-staking/pullRequest?prid=8459503)
```suggestion
            web3.utils.toWei("1000000000"));
```